### PR TITLE
Issue 565 

### PR DESCRIPTION
**Pull request by asharonbaltazar**
**Problem addressed**: 

Dates were squished and the filter was unusable due to screen restraints. 

**Problem fixed:** 

- Hid `input` on smaller screens. 
- Tweaked `h2`s sizes across all screens. 

**Problem hopefully fixed**:

Headers should now sticky on Safari.

### CODE REVIEW

**Comment by csell5** [comment]
@asharonbaltazar looks like you have some conflicts here.

**Comment by asharonbaltazar** [comment]
Something's broken

**Comment by asharonbaltazar** [comment]
Hold on. Don't merge this yet. I want to make one last change.

File: src/components/activities/List.svelte [hunk 1]
```diff
@@ -181,68 +180,49 @@
...
+                    <Card {...activity} editMode="{editMode}" />
+                  </li>
+                {/if}
+              {/each}
+            </ul>
           </div>
-        {/each}
-      </div>
-    {/each}
-  {:else}
```
Comment (csell5) [inline src/components/activities/List.svelte:233]: I don't think you meant to remove this.. Maybe double check this pr.

**Review by csell5 — CHANGES_REQUESTED** [review]
Take another look through this, looks like you're removing a few things you didn't mean too.

**Comment by csell5** [comment]
@asharonbaltazar mind you, I did pull this branch up to latest on master.

**Comment by asharonbaltazar** [comment]
I think it's because of that.

**Review by csell5 — APPROVED** [review]

**Comment by csell5** [comment]
Closes #565

**Pull request merged**